doc/source/cluster/kubernetes/examples/rayserve-llm-example.md

```diff
@@ -0,0 +1,136 @@
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hf-token
+type: Opaque
+stringData:
+  hf_token: <your-hf-access-token-value>
+```
+
+## Step 4: Deploy a RayService
+
+Create a RayService Custom Resource:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray-service.llm-serve.yaml
+```
+
+In this step, a custom Ray Serve Application is setup to serve the `Qwen/Qwen2.5-7B-Instruct` Model, creating an OpenAI-Compatible Server. The source for this example is on [GitHub](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-service.llm-serve.yaml). You can inspect and modify the Serve Config to learn more about the Serve deployment:
+```yaml
+serveConfigV2: |
+  applications:
+  - name: llms
+    import_path: ray.serve.llm:build_openai_app
+    route_prefix: "/"
+    args:
+      llm_configs:
+      - model_loading_config:
+          model_id: qwen2.5-7b-instruct
+          model_source: Qwen/Qwen2.5-7B-Instruct
+        engine_kwargs:
+          dtype: bfloat16
+          max_model_len: 1024
+          device: auto
+          gpu_memory_utilization: 0.75
+        deployment_config:
+          autoscaling_config:
+            min_replicas: 1
+            max_replicas: 4
+            target_ongoing_requests: 64
+          max_ongoing_requests: 128
+```
+
+Wait for the RayService resource to become healthy. You can check its status by running the following command:
+```sh
+kubectl get rayservice ray-serve-llm -o yaml
+```
+
+After a few minutes, the result should be something like this:
+```
+status:
+  activeServiceStatus:
+    applicationStatuses:
+      llms:
+        serveDeploymentStatuses:
+          LLMDeployment:qwen2_5-7b-instruct:
+            status: HEALTHY
+          LLMRouter:
+            status: HEALTHY
+        status: RUNNING
+```
+
+## Step 5: Sending a Request
+
+To send requests to the Ray Serve Deployment, port-forward 8000 port from the Serve application Service:
+```sh
+kubectl port-forward ray-serve-llm-head-svc 8000
+```
+
+Additionally, you can also port-forward 8000 port from the Head application Service to send requests to the serve application.
+
+Keep in mind this Kubernetes Service comes up only after Ray Serve applications are running and ready. This process takes a few minutes after all the pods in the Ray Cluster are up and running.
+
+Test the service with the following command:
+```sh
+curl --location 'http://localhost:8000/v1/chat/completions' --header 'Content-Type: application/json' 
+  --data '{
+      "model": "qwen2.5-7b-instruct",
+      "messages": [
+          {
+              "role": "system", 
+              "content": "You are a helpful assistant."
+          },
+          {
+              "role": "user", 
+              "content": "Provide steps to serve an LLM using Ray Serve."
+          }
+      ]
+  }'
+```
+
```

Comment:
Can you provide an example output? If it is verbose, we can only provide partial of it.